src/view/assign-visually-hidden-style.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+// https://allyjs.io/tutorials/hiding-elements.html
+// Element is visually hidden but is readable by screen readers
+const assignVisuallyHidden = (style: Object) => {
```

Comment:
I prefer my functions not to mutate. Are we avoiding Object.assign?

Reply:
Yes. Object.assign need to be polyfilled. Though in this case much easier to get mutation. I think this is ok for this particular function. object assign polyfill would mutate too.

Reply:
We use spread quite a bit in the library. Wouldn't that be using Object.assign?

Reply:
Spread uses extends helpers which comes from `@babel/runtime` package

Reply:
But it won’t use that if you use object.assign. Can we import extends from Babel runtime and use that for the assign? 🤔

Reply:
My ideal is to not have a function modify the passed in object but rather to return a new object (or to do the Object.assign in place) But I am okay to not get hung up on it